src/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -118,6 +118,11 @@ public CRAMIterator(final SeekableStream seekableStream, final ReferenceSource r
         parser = new ContainerParser(cramHeader.getSamFileHeader());
     }
 
+    public CRAMIterator(final SeekableStream seekableStream, final ReferenceSource referenceSource, final long[] coordinates)
```

Comment:
Can we make this `@deprecated`.  I'd like to push people towards using the one that takes the stringency since it's confusing when they don't.